plugins/woocommerce/src/Admin/PluginsHelper.php

```diff
@@ -317,6 +318,13 @@ public static function install_plugins( $plugins, ?PluginsInstallLogger $logger
 				continue;
 			}
 
+			/**
+			 * Action triggered before a plugin is installed.
+			 *
+			 * @since 9.9
```

Comment:
Since this is now targeting 9.8
```suggestion
			 * @since 9.8
```